client/src/main/java/org/eclipse/hono/client/impl/CommandClientImpl.java

```diff
@@ -210,6 +212,48 @@ protected BufferResult getResult(
         }
     }
 
+    @Override
+    public Future<Void> sendCommandWithTenantScopedResponse(final String command, final Buffer data) {
+        return sendCommandWithTenantScopedResponse(command, null, data, null);
+    }
+
+    @Override
+    public Future<Void> sendCommandWithTenantScopedResponse(final String command, final String contentType, final Buffer data,
```

Comment:
this should be removed, shouldn't it?

Reply:
ups, so sorry. Something went terribly wrong with my local merging.